src/persist/src/file.rs

```diff
@@ -253,6 +301,17 @@ mod tests {
             let instance_dir = temp_dir.path().join(path);
             FileBlob::open(instance_dir.into())
         })
-        .await
+        .await?;
+
+        let temp_dir = tempfile::tempdir().map_err(Error::from)?;
+        blob_impl_test(move |path| {
```

Comment:
let's give MemBlob the same treatment (a flag to enable tombstoneing and a test)